tests/functional/workloads/cnv/test_vm_snapshot_cloning_ops.py

```diff
@@ -79,43 +112,66 @@ def test_vm_snapshot_ops(
         4. Create new vm using restored pvc Verify existing data of the VM are not changed.
         5. Add further data(e.g., new file) to the VM
         6. Repeat the above procedure for all the VMs in the system
-        7. Delete all the VMs created as part of this test
+        7. Stop all the VMs created as part of this test.
```

Comment:
Please add steps for a) expansion and b) verifying the availability of expanded portion for IOs

Reply:
Done